poseidon/poseidonMonitor/endPoint.py

```diff
@@ -0,0 +1,54 @@
+"""
+Created on 2 October 2017
+@author: Jorissss
+"""
+
+import hashlib
+import json
+
+
+
+class EndPoint:
+    def __init__(self, data):
```

Comment:
see write json

Reply:
looks like none of the asserts in test_endpoint_creation are passing circleci

Reply:
@d-grossman The error was that in the test_data the tags were not unicode, json.dumps does (apparently) encode in unicode so one of the to_str()'s came out with u's in front of the tags and one did not.

Reply:
@d-grossman codacy is also complaining about import pytest and the assert statements, anything I can do about that?